src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-top: 0;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you've defined `margin-top` and `margin-right`. It's better to choose one direction for margins to avoid potential margin collapse issues.